core/templates/pages/exploration-editor-page/editor-navigation/editor-navigation.component.html

```diff
@@ -1,5 +1,5 @@
 <div class="navbar-mobile-container">
-  <i class="fa fa-book show-mobile-navbar-icon e2e-test-mobile-options" (click)="toggleMobileNavOptions()" aria-label="Mobile navbar options"></i>
```

Comment:
Why are we dropping this?

Reply:
Initially, lighthouse was giving me an error for that, so I thought aria-labels are not to be given to the elements that are only visible in mobile screeen. But upon checking with other sites, that is not true, so now I have fixed that. PTAL!